Programs/_testembed.c

```diff
@@ -2043,6 +2066,7 @@ struct TestCase
 static struct TestCase TestCases[] = {
     // Python initialization
     {"test_repeated_init_exec", test_repeated_init_exec},
+    {"test_repeated_Py_Initialize", test_repeated_Py_Initialize},
```

Comment:
so far, all function names are lower case. maybe call it the "legacy init"?

Reply:
I called it the "simple initialization API" in the Python test case, so I'll use that name here as well